Vermaat.Crm.Specflow/HelperMethods.cs

```diff
@@ -13,7 +13,10 @@ namespace Vermaat.Crm.Specflow
 {
     public static class HelperMethods
     {
-
+        public static string TryGetAppSettingsValue(string key)
```

Comment:
I don't really see the need of this method. You might aswell just call GetAppSettingsValue(key, true) directly

Reply:
happy to change.  true and false values sometimes confusing have to go to definition to see what that means.